.github/workflows/nsis.yml

```diff
@@ -0,0 +1,68 @@
+name: nsis
+
+on:
+  workflow_run:
+    workflows: [ Build ]
+    types: [ completed ]
```

Comment:
Let me know if I should limit this script running to Releases only.
Personally though, I would recommend having a separate Release workflow that executes after the build and nsis/etc workflows to retrieve artifacts from Release tagged builds and creates a proper release here on GitHub.
That way, we could refactor the online installers to always pull the latest release, or to pull specific releases if needed.

Reply:
I really like the idea of a proper release.

As to whether the script should be limited to releases only, the only reason I could think of for that is size. If one of the installers bundles too much together and pushes us towards some limit, then that could be bad. Otherwise, it'd be nice to click on a specific merge and be able to install it and test it quickly.